configs/mocha.js

```diff
@@ -1,15 +1,12 @@
-module.exports = {
-  plugins: [
-    'mocha'
-  ],
-  env: {
-    mocha: true,
-    node: true
-  },
-  rules: {
-    'mocha/no-exclusive-tests': 2
-  },
-  globals: {
```

Comment:
This is a breaking change that will prevent us from using `expect` in environments such as diagram-js where it is provided externally.

Reply:
Or is this nowadays pulled in / defined by mocha plug-in?

Reply:
This is not defined by mocha plugin ([mocha](https://github.com/lo1tuma/eslint-plugin-mocha/blob/main/index.js#L126), [globals](https://github.com/sindresorhus/globals/blob/HEAD/globals.json#L2075)). I believe we should re-add it.

Reply:
We could also consider adding `sinon` to the globals as we have done in diagram-js: https://github.com/bpmn-io/diagram-js/commit/7de8f7732b2f42f079b6c0c2b2dcb526fbfd78d0
It's commonly used in bpmn.io ecosystem.

Reply:
So mocha, chai, and sinon.

Reply:
Chai can be added via globals: https://github.com/sindresorhus/globals/blob/HEAD/globals.json#L1217

Reply:
We don't need chai, but `expect` only, this is our pattern.